core/domain/fs_domain_test.py

```diff
@@ -15,113 +15,245 @@
 # limitations under the License.
 
 """Tests for filesystem-related domain objects."""
+import logging
+import os
 
+from constants import constants
 from core.domain import fs_domain
+from core.platform import models
 from core.tests import test_utils
 import feconf
 
+app_identity_services = models.Registry.import_app_identity_services()
+(file_models,) = models.Registry.import_models([
+    models.NAMES.file
```

Comment:
could you do split after the ( and then do 
```suggestion
    [models.NAMES.file])
```

Not sure, @apb7 could you confirm which is preferred?

Reply:
I think we prefer the latter, that is, break after the first parenthesis, `(`. Thanks!

Reply:
Done!